src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const ACTIONS = {
+    ADD_PROPERTIES: 'addProperties',
+    REMOVE_PROPERTIES: 'removeProperties',
+    CLEAR: 'clear',
+  };
+
+  for (const action of actions) {
+    switch (action.type) {
+      case ACTIONS.ADD_PROPERTIES:
+        Object.assign(state, action.extraData);
+        break;
+      case ACTIONS.REMOVE_PROPERTIES:
```

Comment:
Add spaces for a better readability 
```suggestion
        break;
        
      case ACTIONS.REMOVE_PROPERTIES:
```